ecosystem/node-checker/vfn-check-client/src/helpers.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{Context, Result};
+use aptos_sdk::types::network_address::NetworkAddress;
+use std::net::{SocketAddr, ToSocketAddrs};
+
+// This function takes a NetworkAddress and returns a string representation
+// of it if it is a format we can send to NHC. Otherwise we return an error.
+pub fn extract_network_address(network_address: &NetworkAddress) -> Result<String> {
```

Comment:
nit: why it's own file if there's only one caller? 😄

Reply:
Smaller files hahah but fair point, I can merge / move more stuff here like you suggested.